lm_eval/models/vllm_causallms.py

```diff
@@ -55,6 +57,7 @@ def __init__(
         gpu_memory_utilization: float = 0.9,
         device: str = "cuda",
         data_parallel_size: int = 1,
+        lora_finetune_path: str = None,
```

Comment:
could we rename this kwarg to something matching VLLM's syntax, or to `peft: str = None` matching the `hf` model type?

Reply:
vLLM's `LoRARequest` uses `lora_local_path` but it may be more intuitive to use `peft` to match hf as you mentioned

Reply:
To clarify, would a HF Hub id of a lora be supported, or are only locally saved loras supported here?

Reply:
Good question. Only locally saved loras are supported by vLLM

Reply:
Got it!

I prefer `lora_local_path` in that case.

Reply:
I agree, made this change